app_dart/lib/src/service/scheduler.dart

```diff
@@ -322,10 +322,14 @@ class Scheduler {
     try {
       final List<Target> presubmitTargets = await getPresubmitTargets(pullRequest);
       final List<Target> presubmitTriggerTargets = getTriggerList(presubmitTargets, builderTriggerList);
```

Comment:
Nit: move these two lines to the `else` logic to skip if they come from revert.

Reply:
Okay will do. I originally left this in just incase to catch formatting errors in the ci.yaml.